LayoutKitTests/ReloadableViewLayoutAdapterCollectionViewTests.swift

```diff
@@ -64,32 +64,32 @@ private class TestCollectionView: LayoutAdapterCollectionView, TestableReloadabl
         reloadDataCount += 1
     }
 
-    private override func insertItems(at indexPaths: [IndexPath]) {
```

Comment:
Not sure why this worked in Xcode 8.0. It doesn't make sense that you could override a method and make it private :)

```
Overriding instance method must be as accessible as its enclosing type
```